views/js/pciCreator/ims/mathEntryInteraction/runtime/mathEntryInteraction.js

```diff
@@ -811,48 +963,57 @@ define([
              * Add the style that sets the width of the gaps, discard previous style
              */
             addGapStyle: function addGapStyle() {
-                var self = this;
-
-                if (self.config.gapStyle) {
-                    self.$container.removeClass(function (index, className) {
+                if (this.config.gapStyle) {
+                    this.$container.removeClass(function (index, className) {
                         return (className.match(/\bmath-gap-[\w]+\b/g) || []).join(' ');
                     });
-                    self.$container.addClass(self.config.gapStyle);
+                    this.$container.addClass(this.config.gapStyle);
+                }
+
+                // in case alternative responses, force the wrap to show
+                const inputWrap = this.$container.find('.math-entry-response-wrap');
+                if (inputWrap.length > 0) {
+                    $(inputWrap[0]).show();
                 }
             },
 
             /**
              * Initialize the PCI :
              * @param {Node} dom
              * @param {Object} config - json
+             * @param {Map} propertyManager
              */
-            initialize: function initialize(dom, config) {
+            initialize: function initialize(dom, config, propertyManager) {
                 this.dom = dom;
                 this.userLanguage = config.userLanguage ? config.userLanguage.replace(/[-_][A-Z].*$/i, '').toLowerCase() : '';
 
                 this.$container = $(dom);
                 this.$toolbar = this.$container.find('.toolbar');
-                this.$input = this.$container.find('.math-entry-input');
-
+                const $input = this.$container.find('.math-entry-input');
+                const id = uid();
+                this.inputs = propertyManager;
```

Comment:
**chore:** The name could better reflect the intent. `propertyManager` is a bit vague 😄